### DISCUSSION TITLE
How to globally import scss in a custom theme?

### DISCUSSION THREAD

**Discussion by jeremeevans** [vuepress/core#757] [Q&A] [answered]
What is the correct way to import scss variables in a custom theme? I would like to have one scss file for my base variables, and have them accessible in scoped style blocks in SFCs that are in my theme. How do I do that?

Thank you!

**Comment by Mister-Hope** [comment, marked as answer]
Hey, you may try `vuepress-plugin-sass-palette`. Here is the docs: https://vuepress-theme-hope.github.io/v2/sass-palette/

It may help your problems.

Also, the `@vuepress/plugin-palette` wrote by meteorlxy can do that too. Try reading docs: https://v2.vuepress.vuejs.org/reference/plugin/palette.html

If you meet troubles, you can continue asking them here, but please be more sepcific.
